plugins/usbpro/UsbSerialPlugin.cpp

```diff
@@ -278,6 +296,11 @@ bool UsbSerialPlugin::SetDefaultPreferences() {
   save |= m_preferences->SetDefaultValue(DEVICE_DIR_KEY, StringValidator(),
                                          DEFAULT_DEVICE_DIR);
 
+  save |= m_preferences->SetDefaultValue(OPENDECK_FPS_LIMIT_KEY,
+                                         UIntValidator(0,
+                                         MAX_OPENDECK_FPS_LIMIT),
```

Comment:
Assuming this doesn't fit:
```suggestion
                                         UIntValidator(0, MAX_OPENDECK_FPS_LIMIT),
```

Please do something like this:
```suggestion
                                         UIntValidator(0,
                                             MAX_OPENDECK_FPS_LIMIT),
```

Reply:
Done